packages/sdk-react/src/components/tags/TagFormModal/TagFormModal.tsx

```diff
@@ -90,7 +95,11 @@ const TagFormModalBase = ({
     const tagCreateMutate = tagCreateMutation.mutate;
     tagCreateMutate(
       {
-        name,
+        header: {
```

Comment:
sorry, I don't get it why should we pass headers here? Can we define headers like in `useUpdateTagMutation`?

Reply:
Nice advice, thank you! Refactored.